libs/perun/dialogs/src/lib/group-sync-detail-dialog/group-sync-detail-dialog.component.ts

```diff
@@ -96,26 +83,6 @@ export class GroupSyncDetailDialogComponent implements OnInit {
     return this.type === 'STRUCTURED';
   }
 
```

Comment:
I think `isBasic` and `isStructured` methods are not used. I think you should remove them